workspaces/theme/plugins/theme/src/utils/createComponents.ts

```diff
@@ -77,36 +77,22 @@ export const createComponents = (themeConfig: ThemeConfig): Components => {
       styleOverrides: {
         root: {
           boxShadow: 'none',
-          backgroundColor: general.cardBackgroundColor,
+          backgroundImage: general.paperBackgroundImage,
           // hide the first child element which is a divider with MuiDivider-root classname in MuiPaper
           '& > hr:first-child[class|="MuiDivider-root"]': {
             height: 0,
           },
         },
-        elevation0: {
-          '& div[class*="Mui-disabled"]': {
-            backgroundColor: 'unset',
-          },
-          '& span[class*="Mui-disabled"]': {
-            backgroundColor: 'unset',
-          },
-          '& input[class*="Mui-disabled"]': {
-            backgroundColor: 'unset',
-          },
-        },
         elevation1: {
           boxShadow: 'none',
-          borderRadius: '0',
-          outline: `1px solid ${general.cardBorderColor}`,
+          outline: `1px solid ${general.paperBorderColor}`,
           '& > hr[class|="MuiDivider-root"]': {
-            backgroundColor: general.cardBorderColor,
+            backgroundColor: general.paperBorderColor,
           },
         },
         elevation2: {
-          backgroundColor: general.tableBackgroundColor,
           boxShadow: 'none',
-          outline: `1px solid ${general.cardBorderColor}`,
-          padding: '1rem',
```

Comment:
Hi @christoph-jerolimov , I added this padding when I worked on [matching table style with PF](https://github.com/janus-idp/backstage-showcase/pull/1155/files#diff-5a8ffc45a7525322869cdf978e8ab33297686196a6c179bf2fcdf2aa18beb40aR106). Is there a way to test out this change with table? Or we can address the issue on styles of `Mui-table`. What do you think?